src/llmcompressor/pytorch/utils/sparsification.py

```diff
@@ -105,15 +105,19 @@ def params_quantized(self) -> int:
         """
         :return: number of parameters across quantized layers
         """
-        return sum(
-            torch.numel(self.trainable_params[f"{name}.weight"])
-            + (
-                torch.numel(self.trainable_params[f"{name}.bias"])
-                if hasattr(layer, "bias") and layer.bias is not None
-                else 0
+        num_params = 0
+        for name, layer in get_quantized_layers(self.module):
+            num_param_weight = torch.numel(
```

Comment:
if getattr(layer, "weight", None) is not None:
    num_params += torch.numel(layer.weight)
if getattr(layer, "bias", None) is not None:
    num_params += torch.numel(layer.bias)